app/components/Settings/EncryptForm/EncryptForm.jsx

```diff
@@ -198,10 +210,9 @@ export default class EncryptForm extends React.Component<Props, State> {
     const { privateKey, passphrase, confirmPassphrase } = this.state
 
     const validInput = this.validate(privateKey, passphrase, confirmPassphrase)
-    if (validInput) {
-      this.setState({ isDisabled: true })
-      onSubmit(privateKey, passphrase, confirmPassphrase)
-      this.setState({ isDisabled: false })
-    }
+
```

Comment:
nitpick: I think this reads a little better

```suggestion
    if (validInput) return onSubmit(privateKey, passphrase, confirmPassphrase)
```